src/Widgets/PricingWidget/PricingWidget.scss

```
@@ -121,7 +121,7 @@
   }
```

Here we also need to watch out. Previously less than 600px used the style of `$screen-sm-max` and only overwrote `widght: 90% !important`. Since the `$screen-sm-max` style is now independent we need to add the missing styles to `600px`:

```suggestion
@media (max-width: 600px) {
  .pricing-widget {
    .pricing-widget-chart {
      float: none !important;
      left: 0px !important;
      top: 0px !important;
      margin: 0 auto;
      width: 90% !important;
      margin-bottom: 25px !important;

      &.last {
        margin-bottom: 0px !important;
      }
    }
  }
}
```